slatedb/src/config.rs

```diff
@@ -600,6 +600,12 @@ pub struct Settings {
     /// The object store cache options.
     pub object_store_cache_options: ObjectStoreCacheOptions,
 
+    /// Preload index/filter metadata into the meta cache on startup. `None` disables preloading.
```

Comment:
Nit: mind adding a note that this works front->back (i.e. it starts with the latest L0 and works downward toward the oldest SST from there)?